proposals/sdk-version-scheme.md

```diff
@@ -0,0 +1,90 @@
+# .NET Core SDK Versioning
+
+.NET Core runtimes and SDKs are used together. They release independently, so it has been a challenge picking sensible version numbers. Going forward, the first and second positions of the version number will match for the Runtime and SDK. The third position will not match since the Runtime and SDK release independently.
+
+The Runtime will follow [semantic versioning](https://semver.org/). The first, second and third positions of the Runtime version number will correspond to major, minor and patch versions, respectively.
+
+The SDK will not follow [semantic versioning](https://semver.org/). The first and second positions of the SDK version number will match the major and minor version of the Runtime it contains. You can use the major.minor version of the SDK to determine the maximum runtime it supports. Each version of the SDK supports previous versions of the runtime. 
+
+For the SDK, both feature-level and servicing changes will be represented in the third position of the version number.
+
+## Scheme
+
+* The first and second position of the version of the .NET Core SDK will always match the .NET Core Runtime.
+* The third position of the version of the .NET Core SDK will generally not match the Runtime.
+* The third position of the .NET Core SDK version will contain gaps to allow servicing of releases.
+
+The third position will increment to the next multiple of 100 for each feature release, and by 1 for each servicing update. As an example of SDK servicing, we may need to update an older version of the SDK to fix an earlier version of the C# compiler.
+
+By matching the major and minor versions between the SDK and Runtime, we will better show how these releases align. In general, programmers can use the highest available SDK, which will be backwards compatible with previous runtimes.
+
+## Examples
+
+A hypothetical set of releases to illustrate the type of pattern:
+
+| Runtime | SDK     | Runtime Change | SDK Change |
+|---------|---------|----------------|------------|
+| 2.1.0   | 2.1.100 | Feature        | Feature    |
+| 2.1.0   | 2.1.200 | None           | Feature    |
+| 2.1.0   | 2.1.201 | None           | Servicing  |
+| 2.1.1   | 2.1.202 | Servicing      | Servicing  |
+| 2.2.0   | 2.2.100 | Feature        | Feature    |
+| 2.2.0   | 2.2.101 | None           | Servicing  |
+
+> Examples:
+>   * SDK feature release: includes C# 7.3
+>   * SDK servicing release: includes .NET Core Runtime security update
```

Comment:
I guess this kind of servicing release refers to one of the type like line 30, where the runtime version is changed as well. Would it be a good idea to also add an example for a release like line 29 where the runtime isn't revved at all?